### PR TITLE
[CHORES] Add handling for CORS and Add Time Limit for Token

### PR DESCRIPTION
**Pull request by JohannesSetiawan**
What changes:
- Adding time limit 30 seconds for email verification and password change token.
- Adding CORS handling in settings.py

### CODE REVIEW

File: authentication/templates/change_password_email_msg.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <h2 style="margin: 0; padding: 10px; border-top-left-radius: 10px; border-top-right-radius: 10px; background-color: #39d69f86; color: #ffffff;">Verify Email</h2>
     <div style="margin: 0; padding: 15px; background-color: #ffffff;">
         <p>Hello,</p>
-        <p>Please copy the token below and enter it in the password recovery forms along with your new password.</p>
+        <p>Please copy the token below and enter it in the password recovery forms along with your new password. The token only valid for 30 seconds.</p>
```
Comment (catherineeangel) [inline authentication/templates/change_password_email_msg.html:5]: why only 30 seconds?

File: revelio/settings_dev.py [hunk 2]
```diff
@@ -81,6 +83,9 @@
 
 WSGI_APPLICATION = 'revelio.wsgi.application'
 
+CORS_ALLOWED_ORIGINS = [
```
Comment (catherineeangel) [inline revelio/settings_dev.py:86]: You can also add our deployed web url

**Review by catherineeangel — CHANGES_REQUESTED** [review]

File: authentication/templates/change_password_email_msg.html (hunk 1 above)
Comment (JohannesSetiawan) [reply authentication/templates/change_password_email_msg.html:5]: Cause i think 3 days is too long. But i think i will change it to 5 minutes

File: revelio/settings_dev.py (hunk 2 above)
Comment (JohannesSetiawan) [reply revelio/settings_dev.py:86]: noted

File: authentication/templates/change_password_email_msg.html (hunk 1 above)
Comment (catherineeangel) [reply authentication/templates/change_password_email_msg.html]: Nice, 5 mins is enough time but not too short

File: revelio/settings_dev.py [hunk 3]
```diff
@@ -85,6 +85,7 @@
 
 CORS_ALLOWED_ORIGINS = [
     "http://localhost:3000",
+    "https://revelio-two.vercel.app",
```
Comment (catherineeangel) [inline revelio/settings_dev.py:88]: Don’t forget to include both staging and production url.

**Review by catherineeangel — CHANGES_REQUESTED** [review]

**Review by catherineeangel — APPROVED** [review]

**Pull request merged**